src/katgpucbf/fgpu/engine.py

```diff
@@ -451,6 +452,7 @@ def __init__(
     ) -> None:
         super().__init__(katcp_host, katcp_port)
         self._populate_sensors(self.sensors)
+        self.sensors.add(DeviceStatusSensor(self.sensors))
```

Comment:
Any particular reason you've `add`ed this here instead of within `_populate_sensors`? (Same question for the XBEngine)

Reply:
I'm with @amishatishpatel on this.